### ISSUE TITLE
Enabling opening Cypress installed locally in 'global' mode

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#889]
Use case is when you have multiple nested projects but a single shared `cypress`.

It's useful to spawn cypress open in global mode so you can pick either project.

```text
/my-project
  package.json
  /node_modules
    /cypress
  /nested-project1
    cypress.json
    /cypress
  /nested-project2
    cypress.json
    /cypress
```

**Comment by brian-mann** [comment]
Fixed by PR https://github.com/cypress-io/cypress/pull/888

**Comment by brian-mann** [comment]
Released in [`1.1.0`](https://docs.cypress.io/guides/references/changelog.html#1-1-0).
